File: src/main/java/com/gmail/borlandlp/minigamesdtools/arena/scenario/simplyteampvp/ExampleScenarioListener.java
package com.gmail.borlandlp.minigamesdtools.arena.scenario.simplyteampvp;

import com.gmail.borlandlp.minigamesdtools.arena.ArenaEventListener;
import com.gmail.borlandlp.minigamesdtools.arena.localevent.ArenaEventHandler;
import com.gmail.borlandlp.minigamesdtools.arena.localevent.ArenaPlayerDeathLocalEvent;
import com.gmail.borlandlp.minigamesdtools.arena.localevent.ArenaPlayerKilledLocalEvent;
import com.gmail.borlandlp.minigamesdtools.arena.scenario.Scenario;


public class ExampleScenarioListener implements ArenaEventListener {
    private Scenario scenario;

    public ExampleScenarioListener(Scenario s) {
        this.scenario = s;
    }

    @ArenaEventHandler
    public void playerDied(ArenaPlayerDeathLocalEvent event) {

    }

    @ArenaEventHandler
    public void playerKilled(ArenaPlayerKilledLocalEvent event) {

    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/arena/scenario/simplyteampvp/ExampleScenarioListener.kt
package com.gmail.borlandlp.minigamesdtools.arena.scenario.simplyteampvp

import com.gmail.borlandlp.minigamesdtools.arena.ArenaEventListener
import com.gmail.borlandlp.minigamesdtools.arena.localevent.ArenaEventHandler
import com.gmail.borlandlp.minigamesdtools.arena.localevent.ArenaPlayerDeathLocalEvent
import com.gmail.borlandlp.minigamesdtools.arena.localevent.ArenaPlayerKilledLocalEvent
import com.gmail.borlandlp.minigamesdtools.arena.scenario.Scenario

class ExampleScenarioListener(private val scenario: Scenario) : ArenaEventListener {
    @ArenaEventHandler
    fun playerDied(event: ArenaPlayerDeathLocalEvent) {}

    @ArenaEventHandler
    fun playerKilled(event: ArenaPlayerKilledLocalEvent) {}
}